app/js/components/LabResultEntry.jsx

```diff
@@ -103,26 +97,79 @@ export class LabResultEntry extends PureComponent {
           </Row>
         </Grid>
       );
-    }
 
-    return (
-      <fieldset>
-        <legend>Result Details:</legend>
-        <div>
-          <EncounterFormPage
-            afterSubmitLink="/"
-            backLink="/"
-            encounterType={ENCOUNTER_TYPES.specimenCollectionEncounterType}
-            formContent={observations}
-            patient={patientHeaderDetail}
-          />
-        </div>
-      </fieldset>
-    );
+      return (
+        <fieldset>
+          <legend>Result Details:</legend>
+          <Row bsClass="row result-range-header">
+            <Col sm={3} />
+            <Col sm={3} />
+            <Col sm={3} />
+            <Col sm={3}>
+              <span className="range-header-text">NORMAL RANGE</span>
+            </Col>
+          </Row>
+          <Row>
+            <EncounterFormPage
+              afterSubmitLink="/"
+              backLink="/"
+              encounterType={ENCOUNTER_TYPES.specimenCollectionEncounterType}
+              formContent={observations}
+              patient={patientHeaderDetail}
+            />
+          </Row>
+        </fieldset>
+      );
+    }
+    return null;
   }
 
-  shouldRedirect() {
-    this.setState({ redirect: true });
+  renderFormContent(member) {
+    const { conceptMembers } = this.props;
+    const memberDetails = conceptMembers[member.uuid];
+    if (memberDetails) {
+      const {
+        hiNormal,
+        lowNormal,
+        lowAbsolute,
+        hiAbsolute,
+        hiCritical,
+        units,
+      } = memberDetails;
+
+
+      const minRange = minValue(lowAbsolute);
+      const maxRange = maxValue(hiAbsolute);
```

Comment:
I think we will need these to be null-safe... that is, a test might not have a hiAbsolute or a lowAbsolute, and in this case we do not want to apply that validation.